templates/splunk-enterprise.template.yaml

```diff
@@ -1,309 +1,267 @@
+    ConstraintDescription: 'Must be a valid IP range in x.x.x.x/x notation.  Use 0.0.0.0/0 for no restrictions.'
+    Description: 'The IP address range that is allowed to connect to the Splunk web interface. Note: a value of 0.0.0.0/0 will allow access from ANY ip address'
     MaxLength: '19'
     MinLength: '9'
     Type: String
   HECClientLocation:
     AllowedPattern: ^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(\/([0-9]|[1-2][0-9]|3[0-2]))$
-    ConstraintDescription: Must be a valid IP range in x.x.x.x/x notation.  Use 0.0.0.0/0
-      for no restrictions.
-    Description: 'The IP address range that is allowed to send data to Splunk HTTP
-      Event Collector. Note: a value of 0.0.0.0/0 will allow access from ANY ip address'
+    ConstraintDescription: 'Must be a valid IP range in x.x.x.x/x notation.  Use 0.0.0.0/0 for no restrictions.'
+    Description: 'The IP address range that is allowed to send data to Splunk HTTP Event Collector. Note: a value of 0.0.0.0/0 will allow access from ANY ip address'
     MaxLength: '19'
     MinLength: '9'
     Type: String
   IndexerInstanceType:
     AllowedValues:
-      - c4.2xlarge
-      - c4.4xlarge
-      - c4.8xlarge
-      - m4.2xlarge
-      - m4.4xlarge
-      - m4.10xlarge
-      - c5.2xlarge
+      - m5.4xlarge
+      - m5.8xlarge
       - c5.4xlarge
       - c5.9xlarge
       - c5.18xlarge
-      - i3.2xlarge
       - i3.4xlarge
       - i3.8xlarge
-    Description: EC2 instance type for Splunk Indexers
-    ConstraintDescription: must be a valid EC2 instance type.
-    Default: c5.4xlarge
+      - i3en.3xlarge
+      - i3en.6xlarge
+      - i3en.12xlarge
+    Description: 'EC2 instance type for Splunk Indexers'
+    ConstraintDescription: 'Must be a valid EC2 instance type.'
+    Default: i3.4xlarge
     Type: String
   SearchHeadInstanceType:
     AllowedValues:
-      - c4.2xlarge
-      - c4.4xlarge
-      - c4.8xlarge
-      - r4.4xlarge
-      - r4.8xlarge
-      - r4.16xlarge
-      - c5.2xlarge
+      - r5.4xlarge
+      - r5.8xlarge
+      - r5.16xlarge
       - c5.4xlarge
       - c5.9xlarge
       - m5.2xlarge
       - m5.4xlarge
+      - m5.8xlarge
       - m5.12xlarge
-    Description: EC2 instance type for Splunk Search Heads
-    ConstraintDescription: must be a valid EC2 instance type.
+    Description: 'EC2 instance type for Splunk Search Heads'
+    ConstraintDescription: 'Must be a valid EC2 instance type.'
     Default: c5.4xlarge
     Type: String
-  IndexerApps:
-    Description: Comma separated list of URLs of Splunk App (or Add-on) tarballs (.spl)
-      to pre-install on indexer(s)
-    Default: ''
-    Type: CommaDelimitedList
-  SearchHeadApps:
-    Description: Comma separated list of URLs of Splunk App (or Add-on) tarballs (.spl)
-      to pre-install on search head(s)
-    Default: ''
-    Type: CommaDelimitedList
   KeyName:
```

Comment:
Please remove single quotes from all the descriptions. Seems like it has got added unintentionally by IDE.